lib/libthr/thread/thr_attr.c

```diff
@@ -375,7 +375,16 @@ _pthread_attr_init(pthread_attr_t *attr)
 	else {
 		/* Initialise the attribute object with the defaults: */
 		memcpy(pattr, &_pthread_attr_default, sizeof(struct pthread_attr));
-
+#ifdef __CHERI_PURE_CAPABILITY__
```

Comment:
Should be be making this conditional on the pure-capability ABI?  We might want to restrict DDC in the hybrid ABI too.